internal/app/admin/http/cache.go

```diff
@@ -210,6 +233,16 @@ func printCacheEntries(keys []string, cache cache.ReadOnlyCache, w http.Response
 	}
 }
 
+func clearCacheEntries(keys []string, o *orchestrator.Orchestrator, w http.ResponseWriter) {
+	errors := orchestrator.Orchestrator.ClearCacheEntries(*o, keys)
+	for _, err := range errors {
+		errMessage, _ := stringify.InterfaceToString(&marshallable.Error{
+			Message: err.Error(),
+		})
+		_, _ = w.Write([]byte(errMessage))
```

Comment:
i dont think this will create the right response json inside a loop. May be create a string based aggreagated response of errors and put the final message into the Message field